.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,14 +25,14 @@ Applicable plugins:
 - [ ] retryFailedStep
 - [ ] screenshotOnFail
 - [ ] selenoid
-- [ ] stepByStepReport
+- [x] stepByStepReport
```

Comment:
Checklist in PR template shouldn't have anything checked. Those changes are done for each individual PR after they are created on github.